README.md

```diff
@@ -12,7 +12,7 @@ MinIO is a high performance distributed object storage server, designed for larg
 
 ### Prerequisites
 
-- Kubernetes version v1.15.5 and above.
+- Kubernetes version v1.17.0 and above for compatibility. MinIO Operator uses `k8s/client-go` v0.17.0.
```

Comment:
we are using v0.18.0 - https://github.com/minio/minio-operator/pull/85/files#diff-37aff102a57d3d7b797f152915a6dc16R15